packages/protocol/test/authorizers/RequesterRrpAuthorizer.sol.js

```diff
@@ -4,8 +4,8 @@ const { expect } = require('chai');
 const utils = require('../utils');
 
 let roles;
-let api3RequesterRrpAuthorizer;
-let airnodeAddress, airnodeId;
+let SelfRequesterRrpAuthorizer;
```

Comment:
that is sort of a different issue, the problem here was that in the old test it didn't matter what airnodeId you used because getRank would be overridden to use bytes32(0). This created a problem because now you would be able to whitelist for any serviceID if you were a superAdmin. This shouldn't be the case because Api3RequesterRrpAuthorizer is designed to be only used with a single serviceID. With the changes to contract this is not possible anymore and Api3RequesterRrpAuthorizer is limited to only a singular serviceID when it comes to whitelisting users. 
This broke the tests and there were 2 options:
- Change the airnodeAddress to address(0) 
- Change the authorizer to SelfRequesterRrpAuthorizers

I opted for the second one because it made more sense since it has reference to airnode address.